ConceptMap/ConceptMap-Genomics-procedure-transfusiontype.json

```diff
@@ -0,0 +1,77 @@
+{
+  "resourceType": "ConceptMap",
+  "id": "genomics-procedure-transfusiontype",
+  "url": "https://fhir.nhs.uk/Conceptmap/genomics-procedure-transfusiontype",
+  "version": "0.1.0",
+  "name": "GenomicsProcedureTransfusionType",
+  "title": "Genomics Procedure Transfusion Type",
+  "status": "draft",
+  "date": "2024-07-12T00:00:00.000Z",
+  "publisher": "NHS England",
+  "contact": [
+    {
+      "name": "NHS England",
+      "telecom": [
+        {
+          "system": "email",
+          "value": "interoperabilityteam@nhs.net"
+        }
+      ]
+    }
+  ],
+  "description": "A ConceptMap used for the translation and recording of concepts from MDS source data(code-options) to equivalent Snomed CT codes, and FHIR spec.",
+  "purpose": "ConceptMap used for the translation and recording of concept for type of transfusion procedure that a patient may undergo. .i.e, value for procedure.code field ",
+  "copyright": "Copyright © 2023+ NHS England Licensed under the Apache License, Version 2.0 (the \\\"License\\\"); you may not use this file except in compliance with the License. You may obtain a copy of the License at  http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an \\\"AS IS\\\" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License. HL7® FHIR® standard Copyright © 2011+ HL7 The HL7® FHIR® standard is used under the FHIR license. You may obtain a copy of the FHIR license at  https://www.hl7.org/fhir/license.html.",
+  "targetUri": "http://snomed.info/sct?fhir_vs=ecl/%3C%3C%205447007%20",
+  "group": [
+    {
+      "target": "http://snomed.info/sct",
+      "element": [
+        {
+          "code": "packed-red-blood-cells",
+          "display": "Packed Red Blood Cells",
+          "target": [
+            {
+              "code": "71493000",
+              "display": "Transfusion of packed red blood cells (procedure)",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "plasma",
+          "display": "Plasma",
+          "target": [
+            {
+              "code": "13569004",
+              "display": "Transfusion of plasma (procedure)",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "platelets",
+          "display": "Platelets",
+          "target": [
+            {
+              "code": "12719002",
+              "display": "Platelet transfusion (procedure)",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "unknown",
+          "display": "UNKNOWN",
+          "target": [
+            {
+              "code": "261665006",
```

Comment:
Here we might need to replace the concept with the general 5447007 | Transfusion (procedure) | code (changing the equivalence to something like specializes). I just want to make sure that even if the transfusion type is unknown, the general Procedure.code for transfusion is used, rather than a code for unknown